### PR TITLE
ui: enhance share link dialog

### PR DESCRIPTION
**Pull request by Chilledheart**
screen shot 2015-06-01 at 3 02 22 pm
screen shot 2015-06-01 at 3 03 28 pm

### CODE REVIEW

**Comment by shoeper** [comment]
is it really "Download Only"?

You mean "Direct Download", don't you?

**Comment by Chilledheart** [comment]
@shoeper Yeah, you're correct.

File: src/filebrowser/sharedlink-dialog.cpp [hunk 1]
```diff
@@ -16,18 +17,25 @@ SharedLinkDialog::SharedLinkDialog(const QString &text, QWidget *parent)
...
+    editor_->selectAll();
+    editor_->setReadOnly(true);
+    layout->addWidget(editor_);
 
     QHBoxLayout *hlayout = new QHBoxLayout;
 
+    QCheckBox *is_download_checked = new QCheckBox(tr("Direct Download"));
+    connect(is_download_checked, SIGNAL(stateChanged(int)),
+            this, SLOT(onDownloadStateChanged(int)));
+    is_download_checked->setFont(changeFontSize(is_download_checked->font(), 10));
```
Comment (lins05) [inline src/filebrowser/sharedlink-dialog.cpp]: Why not set the font size in style sheet?

Comment (Chilledheart) [inline src/filebrowser/sharedlink-dialog.cpp]: It costs at least two lines of code to set font size in style sheet (`QObject::setObjectName` and `font-size:xxx`), while it costs only one loc (line of code) to set it in `C++`.

Comment (Chilledheart) [inline src/filebrowser/sharedlink-dialog.cpp]: @lins05 I made some changes hoping the code is more clear.

**Comment by lins05** [comment]
The ok button should be "OK" instead of "Ok".

**Pull request merged**